cdhweb/pages/management/commands/exodus.py

```diff
@@ -238,6 +285,32 @@ def form_pages(self):
         'root': Collection.objects.get(pk=get_root_collection_id())
     }
 
+    def profile_pages(self):
+        """Exodize all existing Profiles to new ProfilePage model."""
+        # all the fields on Profile that moved to Person will have been handled
+        # by django migrations; we just need to create new Page models
+        people_landing = PeopleLandingPage.objects.first()
+        if not people_landing:
```

Comment:
should this warn or error? it shouldn't happen, since we create it first, but seems like we would want to know if it didn't run

Reply:
the reason this check is here is for when the database has a fixture loaded but is otherwise empty, like when you need to exodize an old fixture so that it can be used with wagtail. it's legit for there to not be a people landing page in such cases, but we do want to proceed and do everything else we can